ui/apps/platform/src/routePaths.ts

```diff
@@ -306,7 +306,7 @@ const routeRequirementsMap: Record<RouteKey, RouteRequirements> = {
     },
     'workload-cves': {
         featureFlagDependency: ['ROX_VULN_MGMT_WORKLOAD_CVES'],
-        resourceAccessRequirements: everyResource(['Deployment', 'Image', 'WatchedImage']),
+        resourceAccessRequirements: everyResource(['Deployment', 'Image']),
```

Comment:
Lack of permissions to the `WatchedImage` resource should not block the Workload CVEs page, so this is removed. Adding more granular UI checks for this will be handled separately in ROX-19968